new/LIT7507HymnBetaMeng.xml

```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7507HymnBetaMeng" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ኦእግዝእትየ፡ ማርያም፡ እምቤተ፡ መንግሥት፡ ወእምቤተ፡ ክህነት፡ ተወለድኪ፡ </title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">ʾO-ʾǝgzǝʾtǝya Māryām ʾǝm-beta mangǝśt wa-ʾǝm-beta kǝhǝnat tawaladki</title>
+                <title xml:lang="en" corresp="#t1">O my Lady Mary, you were born from the house of Kingship and the house of Priesthood...</title>
+                <title xml:lang="en" xml:id="t2">Hymn to Virgin Mary</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Poetry"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MK" when="2025-06-06">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <div type="textpart" subtype="incipit">
+                    <ab>ኦእግዝእትየ፡ ማርያም፡ እምቤተ፡ መንግሥት፡ ወእምቤተ፡ ክህነት፡ ተወለድኪ፡ እንተ፡ ዘወለድኪ፡ ንጉሠ፡፡</ab>
```

Comment:
It would be good to include a `<note>` in which you state the source for the text (and a `<label>` also can be useful :) )